### ISSUE TITLE
About the answer for the section, Links and Voting, of the graphql-python tutorial

### ISSUE THREAD

**Issue by vincenterc** [howtographql/howtographql#993]
The answer for the section, Links and Voting, of the graphql-python tutorial is "Input".
[https://www.howtographql.com/graphql-python/5-links-and-voting/](https://www.howtographql.com/graphql-python/5-links-and-voting/)

But in this section we use class Arguments not Input, I think the answer should be Arguments.

**Comment by christopherthompson81** [comment]
I agree, the question and its answer aren't covered until the much later "relay" section of the tutorial. The question should reflect something that was learned during the Links and Voting section of the tutorial.

**Comment by tbrlpld** [comment]
The phrasing of the question should also be adjusted: 

"In which Python class is defined the arguments for a Mutation?"

-> 

"In which Python class are the arguments for a Mutation defined?"
